source/manage/cloud-ip-filtering.rst

```diff
@@ -0,0 +1,80 @@
+Cloud IP Filtering
+========================
+
+.. include:: ../_static/badges/ent-cloud-only.rst
+   :start-after: :nosearch:
+
+
+IP filtering is a powerful security feature that allows system admins to control access to their workspace by defining approved IP ranges. Only users within these specified IP ranges can access the workspace, ensuring enhanced security for your workspace. IP filtering requires a subscription to Mattermost Cloud Enterprise.  
```

Comment:
I removed capitalization on ``filtering`` in ``IP Filtering`` in this PR but if we want the docs to treat this as an official product feature name, happy to bring the capitalization back.

Reply:
I'm 0/5. Spoke with @spirosoik and he's fine with lowercase as well.